server/channels/app/platform/web_hub_test.go

```diff
@@ -639,8 +675,15 @@ func BenchmarkHubConnIndex(b *testing.B) {
 	b.ResetTimer()
 	b.Run("Add", func(b *testing.B) {
 		for i := 0; i < b.N; i++ {
-			connIndex.Add(wc1)
-			connIndex.Add(wc2)
+			err := connIndex.Add(wc1)
+			if err != nil {
+				return
+			}
```

Comment:
returning here could lead to an invalid benchmark. Let's just check the error. `require.NoError(t, err)` works in benchmarks, too